src/content/docs/logs/forward-logs/kong-gateway.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Forward Kong Gateway Logs
+tags:
+  - Logs
+  - Enable log management in New Relic
+  - Enable log monitoring in New Relic
+
+metaDescription: How to forward your logs to New Relic using our kong Gateway file log plugin, so you can use enhanced log management capabilities.
+
+freshnessValidatedDate: never
+---
+
+Forwarding Kong Gateway logs to New Relic provides valuable insights into API gateway performance, health, security, and usage. This document outlines the process of using the File Log plugin to forward logs from Kong Gateway to New Relic.
+
+## Prerequisites
+
+* Kong Gateway with installed [Custom Resource Definitions (CRDs)] (https://docs.konghq.com/kubernetes-ingress-controller/latest/plugins/rate-limiting/#install-the-gateway-apis)
```

Comment:
@gunnalahema Are there other compatibility requirements around OS, versioning, and so on?

Reply:
No